content/02.results.md

```diff
@@ -0,0 +1,77 @@
+## Population Sampling
+
+DNA extracted from wild-caught _Anopheles_ mosquitoes were submitted to the Ag1000G consortium in 23 sets by consortial partners. (chris' line)
+
+## Whole Genome Sequencing and Alignment
+
+A total of 4,693 individual mosquitoes were sequenced on either Illumina HiSeq2000 (n=3,130) or Illumina HiSeqX (n=1,563) to a target coverage of 30X.
+
+Between machine types the median number of bases sequenced per sample was 9.76Gb and 10.33Gb respectively, representing a difference in yield (two-tailed mann-whitney U p < 0.0001).
+
+These values correspond to a yield per reference base (vs AgamP4) of 35.76X and 37.82X. 
+
+91.9% of HiSeqX runs and 80.5% of HiSeq2000 runs met the target yield of 30X.
+
+Reads were aligned to the AgamP4 reference genome using `bwa` version `0.7.15`.
+
+Indel realignment was performed using GATK `v3.7-0` RealignerTargetCreator and IndelRealigner. 
+
+Single nucleotide polymophisms were called against AgamP4 using GATK UnifiedGenotyper `v3.7-0`.
+
+Sample genotypes were called independently, in genotyping mode, given all possible alleles at each site, allowing parallelisation over samples.
+
+Coverage considered at individual sites was capped at 250.
+
+Full details of pipelines including all parameter settings are provided in supplementary.
+
+All samples successfully completed the pipeline and entered the sample quality control (QC) process.
+
+### Sample QC
+
+The sample QC process was composed of three stages, sequence quality assurance, replicate handling, and anomaly detection.
+
+668 samples were removed where sequencing was of insufficient quality to accurately call genotypes across the whole genome.
+
+Exclusions were due to poor coverage (n=410), potential contamination (n=229), and the autosomal vs X coverage ratio not following the expected bimodal distribution (n=29).
+
+Where technical replicates were available, we excluded 4 pairs (8 samples) with low genotype concordance. 
+
+Where pairs met the concordance threshold we excluded the lower quality sample.
+
+In total 407 samples in were excluded in favour of better quality samples, based on skewedness of the mean vs median.
```

Comment:
"mean vs median" what? Also, not sure how you'd like this reviewed, but I think vs should be written _vs._